common/config.gni

```diff
@@ -24,6 +24,8 @@ declare_args() {
   flutter_prebuilt_dart_sdk = false
 }
 
+is_aot = flutter_runtime_mode == "profile" || flutter_runtime_mode == "release"
```

Comment:
This doesn't hold for Fuchsia. I don't think we should add this to the common config if its not actually true everywhere.